cmd/lnmuxd/run.go

```diff
@@ -324,7 +324,6 @@ func initLndClients(ctx context.Context, cfg *LndConfig) ([]lnd.LndClient, *chai
 			Logger:       log,
 			PubKey:       pubkey,
 			Network:      network,
-			Timeout:      cfg.Timeout,
```

Comment:
Would users find it useful to make timeouts configurable

Reply:
I thought they would not. It would also be two timeouts to configure after this pr. What do you think?

Reply:
I don't think that will be very useful.
If in the future people want it to be configurable, it will be an easy change to make anyway.